sdk/loadtesting/azure-developer-loadtesting/swagger/README.md

```diff
@@ -25,7 +25,14 @@ directive:
   - from: swagger-document
     where: $["paths"]["/test-runs/{testRunId}/metric-definitions"].get
     transform: $["operationId"] = "LoadTestRun_GetMetricDefinitions";
-  
+  - from: swagger-document
+    where: $["paths"]["/test-runs/{testRunId}/metric-dimensions/{name}/values"].get
+    transform: >
+      $["parameters"][5]["x-ms-client-name"] = "timeInterval";
+  - from: swagger-document
+    where: $["paths"]["/test-runs/{testRunId}/metric-dimensions/{name}/values"].get
+    transform: >
+      $["parameters"][3]["x-ms-client-name"] = "metricName";
```

Comment:
@NiveditJain How about change swagger directly so that all language can apply for the name change?

Reply:
Our swagger is already in GA. Now it would be breaking change in REST API and SDKs

Reply:
I'm not sure why we updated these two parameter names for `list_metric_dimension_values`, but not in `list_metrics`?
Also, now in the same method, we have both `interval` and `time_interval` parameters - is this intentional? Was this feedback received for one of the other languages?

Reply:
regarding `time_interval` actually we recieved a feedback from @tg-msft to rename `timeSpan` to `timeInterval`
![image](https://user-images.githubusercontent.com/40313233/221735672-08a1c8b6-a5f0-47d6-951c-230f2fe4c9be.png) here: https://apiview.dev/Assemblies/Review/4200a804f382479bb4d2bb4a3323b8c7

Thanks for pointing out that I missed for `list_metrics`, will do the same changes.

Reply:
@annatisch this issue is fixed in the latest commit and added to the same APIView